src/mscorlib/src/System/Text/DecoderBestFitFallback.cs

```diff
@@ -34,13 +31,7 @@ public override DecoderFallbackBuffer CreateFallbackBuffer()
         }
 
         // Maximum number of characters that this instance of this fallback could return
-        public override int MaxCharCount
-        {
-            get
-            {
-                return 1;
-            }
-        }
+        public override int MaxCharCount => 1;
```

Comment:
I'm working on removing all the code that BCLRewriter strips, and that's partly complete. I figured that would make the unification work easier since just about all of that should also be missing from CoreRT.

I expect to have someone work on the unification soon after I've done that so that UWP work can proceed.